sovrin_client/test/cli/test_nym.py

```diff
@@ -1,14 +1,22 @@
 import pytest
+from plenum.common.signer_did import DidSigner
 from plenum.common.signer_simple import SimpleSigner
 from sovrin_client.client.wallet.wallet import Wallet
-from sovrin_client.test.cli.helper import prompt_is, addNym
+from sovrin_client.test.cli.helper import prompt_is, addNym, ensureConnectedToTestEnv
 from sovrin_common.roles import Roles
-from sovrin_node.test.did.conftest import wallet, abbrevVerkey, abbrevIdr
+from sovrin_node.test.did.conftest import wallet, abbrevVerkey
+
+TRUST_ANCHOR_SEED = b'TRUST0NO0ONE00000000000000000000'
 
 
 @pytest.fixture("module")
-def trustAnchorSigner():
-    return SimpleSigner()
+def trust_anchor_did_signer():
+    return DidSigner(seed=TRUST_ANCHOR_SEED)
+
+
+@pytest.fixture("module")
+def trust_anchor_cid_signer():
```

Comment:
Plenum still supports them. So testing them still makes sense. I've changed the CLI to guild you to use DID instead and will only generate key-identifiers using DID but the NYM transaction still supports them so testing that capability still makes sense.